### ISSUE TITLE
setting-up fail2ban

### ISSUE THREAD

**Issue by gagou9** [thewca/worldcubeassociation.org#41]
Fail2ban will block IPs trying to do weird stuff on our ports.
It will use iptables to block them, and will unblock after a time. (or not)

**Comment by ronaldmansveld** [comment]
Likely no longer relevant as we run on AWS now

**Comment by jfly** [comment]
Also still relevant in that anyone on earth can *try* ssh-ing to our server, and that may not be great. However, I stand by everything I said in https://github.com/thewca/worldcubeassociation.org/issues/37#issuecomment-429097624, and agree with closing this issue.
